scripts/hudson/narayana.sh

```diff
@@ -543,7 +551,6 @@ function rts_tests {
 
 function lra_tests {
   echo "#0. LRA Test"
-  ./build.sh install -pl ArjunaCore/arjunacore,rts -am -DskipTests -Pcommunity "$@"
```

Comment:
I think this should be a separate commit to the WFLY changes but I think it's fair to do it (so long as NARAYANA_BUILD=1 is set for the LRA profile - which I see it is proposed to be)